common/ntp/ntp.go

```diff
@@ -0,0 +1,35 @@
+package ntp
+
+import (
+	"time"
+
+	beevik_ntp "github.com/beevik/ntp"
+)
+
+const (
+	// tolerance range of local clock with NTP time server
+	toleranceRange = time.Duration(50 * time.Millisecond)
+)
+
+// IsLocalTimeAccurate returns whether the local clock acturate or not
+func IsLocalTimeAccurate(ntpServer string) bool {
```

Comment:
For most of the cases, returning an error instead of boolean would be more preferable since it gives more context about what error is about. For this part maybe returning error like 

```
errors.Wrapf("system clock is off by %v", diff)
```

will be more resourceful.

Reply:
added error